src/tests/components/AddItem.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Button, TextInput } from 'react-native';
+import { shallow, mount } from 'enzyme';
+import AddItem from '../../components/AddItem';
+
+const mockAddItem = jest.fn();
+
+describe('AddItem component', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('has three text input fields', () => {
+    const wrapper = mount(<AddItem addItem={mockAddItem} />);
```

Comment:
Since all the tests have wrapper in common, can we add this to a `beforeEach()`?

Reply:
fixed! -- for the three text input fields test, the wrapper is actually a mount and not a shallow but I just did `wrapper = mount...`

Reply:
Ohh, why is the wrapper for the three text input fields test (the one on L13) a mount, not a shallow? The test doesn't look like it's doing anything different from the others. 🤔

Reply:
uh please ignore the mount, when I was testing it, it failed but now it's.. passing?